src/app/model.elm

```diff
@@ -1,17 +1,13 @@
-module App.Model exposing (Answer, Model, Poll)
+module App.Model exposing (Model, Poll)
 
-
-type alias Answer =
-    { id : String
-    , answer : String
-    , votes : Int
-    }
+-- TODO: define type alias for Answer here with id and answer as strings and votes as a number.
```

Comment:
Too much details on the Todo I think, you can let them going to http://api.alexrieux.fr/ to look at the documentation and figure out what are the types for Answer